.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        emacs-version:
+          - 28.2
+          - 29.4
+          - 30.1
+        experimental: [false]
+        include:
+        - os: ubuntu-latest
+          emacs-version: snapshot
+          experimental: true
+        - os: macos-latest
+          emacs-version: snapshot
+          experimental: true
+        - os: windows-latest
+          emacs-version: snapshot
+          experimental: true
+
+    steps:
+    - uses: actions/checkout@v4
+
+    - uses: jcs090218/setup-emacs@master
+      with:
+        version: ${{ matrix.emacs-version }}
+
+    - uses: emacs-eask/setup-eask@master
+      with:
+        version: 'snapshot'
+
+    - name: Build
+      run: |
+        eask package
+        eask install
+        eask compile
+
+    - name: Run linter
+      run: |
+        eask lint checkdoc
```

Comment:
It looks like it did find some doc issues, but the CI still counted as a pass.  Is there a way to surface those issues (if we even want to?)

<img width="1438" alt="Screenshot 2025-06-24 at 8 31 10 AM" src="https://github.com/user-attachments/assets/a3405ef8-788b-4791-ba8e-6f39ef2a5637" />

Reply:
We can use `--strict` flag but I usually have it optional. 🤔 Do you want to make it fail?

Reply:
Hm.  Probably not.  Checkdoc be annoying sometimes.